src/base/types/Reaction.jl

```diff
@@ -1,3 +1,4 @@
+using Base: Float64
```

Comment:
Why this?

Reply:
I took this out, I didn't add it, no idea how it got into master to begin with

Reply:
but this PR is adding it?

Reply:
Nope, it should not be added. I think VScode is automatically adding things like this.. It is very irritating :/ I deleted it